packages/contracts/test/voting/token-voting.ts

```diff
@@ -121,11 +152,16 @@ describe('TokenVoting', function () {
           votingSettings,
           governanceErc20Mock.address
         )
-      ).to.be.revertedWith(ERRORS.ALREADY_INITIALIZED);
+      ).to.be.revertedWith('Initializable: contract is already initialized');
     });
   });
 
   describe('Proposal creation', async () => {
+    beforeEach(async () => {
+      await setBalances([{receiver: signers[0].address, amount: 1}]);
+      await setTotalSupply(1);
```

Comment:
unecessary call because the `totalSupply` would already be 1 after `setBalances`

Reply:
Same as here: https://github.com/aragon/core/pull/178#discussion_r1065773796